.github/workflows/release-changelog.yml

```diff
@@ -0,0 +1,49 @@
+name: Changelog vs Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
For me it looks like a redundant condition. 

It's only triggered by a tag https://github.com/MikeOrtiz/TouchImageView/pull/337/files#diff-305377beb31aab4ccc3ffbe29a11cd428b5a52460bc855ceba49c92faec46bf3R5

Reply:
Removed